README.md

```diff
@@ -586,6 +587,38 @@ func main() {
 }
 ```
 
+#### Values from Files
+
+You can also have the default value set from file via `FilePath`.  e.g.
+
+<!-- {
+  "args": ["&#45;&#45;help"],
+  "output": "language for the greeting.*APP_LANG"
```

Comment:
`gfmrun` expects the output specified here to match the program output when ran (this helps make the examples are not invalidated by code changes). In this particular case, I might suggest writing a temporary file in the example and then having a default action on `app` that prints out the value, updating this expected `output` to match.

You can test locally via:

```
go get github.com/urfave/gfmrun/...
./runtests gfmrun
```